core/field_variable.js

```diff
@@ -32,6 +32,8 @@ goog.require('Blockly.Variables');
 goog.require('goog.asserts');
 goog.require('goog.string');
 
+var RENAME_VARIABLE_ID = 'RENAME_VARIABLE';
```

Comment:
More seriously, these should probably be constants in Blockly.Constants, and they should definitely have 'ID' in the text.